Enable News Feed feature flag

jrainville opened the pull request:
Ready for merge once https://github.com/status-im/status-go/pull/6586 is merged

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | ed6272f9 | [#1](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-17830/1/) | 2025-04-22 20:58:23 | ~8 min | `tests/nim` | [:page_facing_up:`log`](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-17830/1//consoleText) |

jrainville commented:
I just rebased this PR and also updated the status-go branch to have the test Desktop RSS feed. So you can just post News there and see them show up (after 30 minutes or after a restart). 

Ping me in you want to have the link and be added as an editor
